src/style.css

```diff
@@ -0,0 +1,104 @@
+* {
+  outline: 0 !important;
+}
```

Comment:
don't use !important unless it's necessary, you should only reset outline for your input
```suggestion
```

Reply:
done